.circleci/docker/ubuntu/Dockerfile

```diff
@@ -44,6 +44,10 @@ ARG GCC_VERSION
 ADD ./common/install_gcc.sh install_gcc.sh
 RUN bash ./install_gcc.sh && rm install_gcc.sh
 
+# Install lcov for C++ code coverage
+ADD ./common/install_lcov.sh install_lcov.sh
```

Comment:
do we need to use a flag to enable/disable it

Reply:
@malfet suggested adding by default; it is 100Kb anyway